src/Sulu/Bundle/AdminBundle/Gruntfile.js

```diff
@@ -121,25 +137,44 @@ module.exports = function(grunt) {
                 files: [
                     {src: ['Resources/public/dist/app.min.js'], dest: 'Resources/public/dist/app.min.js'}
                 ]
+            },
+            buildLogin: {
+                options: {
+                    variables: {
+                        '/js/login.js': '/dist/login.js'
+                    },
+                    prefix: ''
+                },
+                files: [
+                    {src: ['Resources/public/dist/login.min.js'], dest: 'Resources/public/dist/login.min.js'}
+                ]
             }
         },
         requirejs: {
-            compile: {
+            app: {
                 options: {
                     baseUrl: 'Resources/public/js/',
                     mainConfigFile: 'Resources/public/js/main.js',
                     preserveLicenseComments: false
                 }
+            },
+            login: {
```

Comment:
I use the husky.js in both cases, so it gets cached. the entry points for the login and the admin are in the login.js and the app.js which are both quite small. Moreover i can't use the app.js as an entry point for the login as i need to do different things when starting the app. I'd have to put the things i do now in the login.js into the app.js with an if statement which is not really nice concerning the code style